### ISSUE TITLE
Enforce License Compliance workflow fails

### ISSUE THREAD

**Issue by aminvakil** [getsentry/self-hosted#1351]
### Version

irrelevant

### Steps to Reproduce

Open a PR in self-hosted.

### Expected Result

Pass FOSSA Scan.

### Actual Result

https://github.com/getsentry/self-hosted/runs/5343799554?check_suite_focus=true
```
Run fossas/fossa-action@v1.1.0
  with:
    run-tests: false

/home/runner/work/_actions/fossas/fossa-action/v1.1.0/webpack:/fossa-action/node_modules/@actions/core/lib/core.js:94
        throw new Error(`Input required and not supplied: ${name}`);
^
Error: Input required and not supplied: api-key
    at Object.getInput (/home/runner/work/_actions/fossas/fossa-action/v1.1.0/webpack:/fossa-action/node_modules/@actions/core/lib/core.js:94:1)
    at Object.373 (/home/runner/work/_actions/fossas/fossa-action/v1.1.0/webpack:/fossa-action/src/config.ts:3:1)
    at __nccwpck_require__ (/home/runner/work/_actions/fossas/fossa-action/v1.1.0/webpack:/fossa-action/webpack/bootstrap:20:1)
    at Object.144 (/home/runner/work/_actions/fossas/fossa-action/v1.1.0/webpack:/fossa-action/src/index.ts:3:1)
    at __nccwpck_require__ (/home/runner/work/_actions/fossas/fossa-action/v1.1.0/webpack:/fossa-action/webpack/bootstrap:20:1)
    at /home/runner/work/_actions/fossas/fossa-action/v1.1.0/webpack:/fossa-action/webpack/startup:4:1
    at Object.<anonymous> (/home/runner/work/_actions/fossas/fossa-action/v1.1.0/dist/index.js:5257:12)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
```

**Comment by chadwhitacre** [comment]
Hrm, also failed in `sentry-docs` ... disabled for now until I can debug.

**Comment by chadwhitacre** [comment]
https://github.com/getsentry/sentry-docs/pull/4769

**Comment by aminvakil** [comment]
@chadwhitacre I think this has been fixed as https://github.com/getsentry/self-hosted/pull/1347 has been merged, right?

**Comment by aminvakil** [comment]
Fixed by #1347.
